cluster/manifests/skipper/daemonset.yaml

```diff
@@ -55,6 +54,7 @@ spec:
           - "-enable-connection-metrics"
           - "-oauth2-tokeninfo-url={{ .ConfigItems.tokeninfo_url }}"
           - "-histogram-metric-buckets=.01,.025,.05,.075,.1,.2,.3,.4,.5,.75,1,2,3,4,5,7,10,15,20,30,60,120,300,600"
+          - "-opentracing=lightstep component-name=skipper-ingress token={{ .ConfigItems.lightstep_token }} collector=tracing.zmon.zalan.do:8444 cmd-line=skipper-ingress max-buffered-spans=4096"
```

Comment:
The token should be put in a secret. You could mount it as a environment variable and use the `$(TOKEN)` reference in the flag.

Reply:
no it can't be done like this

Reply:
Nono, I mean you can use `$(TOKEN)` where you define the flag. Kubernetes should infer this when starting the pod.